### PR TITLE
[10.0] backport session summary report

### PR DESCRIPTION
**Pull request by PierrickBrun**
This is a backport of the POS Session Summary report removed in 10.0.

ref : https://github.com/odoo/odoo/commit/e741de8b0c152279eb2d20faa77d06df8e50b91a

### CODE REVIEW

**Comment by pedrobaeza** [comment]
We have in 9.0 the module https://github.com/OCA/pos/tree/9.0/pos_session_summary. Is it the same?

**Comment by PierrickBrun** [comment]
No, they serve different purpose. This one adds a report and pos_session_summary adds computed fields to the pos.session.

File: pos_report_sessionsummary/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'POS report Session Summary',
+    'version': '10.0.0',
```
Comment (chienandalu) [inline pos_report_sessionsummary/__manifest__.py]: 10.0.1.0.0

File: pos_report_sessionsummary/README.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
```
Comment (chienandalu) [inline pos_report_sessionsummary/README.rst]: Use the new OCA readme system: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: pos_report_sessionsummary/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    'summary': 'Backport of the report from 9.0',
+    'author': 'Akretion, Odoo SA',
+    'depends': ['point_of_sale'],
+    'data': [
+        'sessionsummary_report.xml',
+        'views/report_sessionsummary.xml',
+    ],
+    'installable': True,
+    'application': False,
+    'website': 'https://www.odoo.com/page/point-of-sale',
```
Comment (chienandalu) [inline pos_report_sessionsummary/__manifest__.py]: https://github.com/OCA/pos

File: pos_report_sessionsummary/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```
Comment (chienandalu) [inline pos_report_sessionsummary/__manifest__.py]: Add Odoo copyright, but also OCA's and proper licence

**Review by chienandalu — CHANGES_REQUESTED** [review]
Shouldn't it be named pos_report_session_summary better?

**Comment by PierrickBrun** [comment]
Thanks for the feedback !
Requested changes are done, PR is still red because some other modules have linting errors. No errors detected on this one.

**Review by bealdav — APPROVED** [review]

File: pos_report_session_summary/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# copyright Odoo SA
+# copyright OCA
+# Licence : LGPL-3
+
+{
+    'name': 'POS report Session Summary',
+    'version': '10.0.1.0.0',
+    'category': 'Point Of Sale',
+    'summary': 'Backport of the report from 9.0',
```
Comment (legalsylvain) [inline pos_report_session_summary/__manifest__.py]: I think it's not a good summary for a module. The summary is better a short resume of the module.

**Review by legalsylvain — COMMENTED** [review]
A minor change required.
Otherwise LGTM For me. Thanks for this work @PierrickBrun.

File: pos_report_session_summary/session_summary_report.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<odoo>
+    <data>
```
Comment (Daemo00) [inline pos_report_session_summary/session_summary_report.xml]: Pleas remove `data` tag in compliance with https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0#tasks-to-do-in-the-migration

File: pos_report_session_summary/views/report_session_summary.xml [hunk 7]
```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data>
```
Comment (Daemo00) [inline pos_report_session_summary/views/report_session_summary.xml]: Remove `data` tag here too

File: pos_report_session_summary/views/report_session_summary.xml [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+<template id="report_sessionsummary">
```
Comment (Daemo00) [inline pos_report_session_summary/views/report_session_summary.xml]: Missing underscore: `report_session_summary`

**Review by Daemo00 — CHANGES_REQUESTED** [review]

**Comment by Daemo00** [comment]
Just reviewed code, only the underscore modification is really required (otherwise I think this won't work)

**Comment by legalsylvain** [comment]
@chienandalu, @Daemo00, could you update your review ? 
kind regards.

**Review by Daemo00 — APPROVED** [review]
Code review and tried on runbot, OK for me.

**Comment by hparfr** [comment]
Hi, is it mergeable now ?

**Comment by legalsylvain** [comment]
Yes thanks.

**Pull request merged**